stdci.yaml

```diff
@@ -3,9 +3,14 @@ sub-stages:
   - k8s-1.10.4-release
   - k8s-1.11.0-dev
   - k8s-1.11.0-release
+<<<<<<< HEAD
```

Comment:
Looks like an unresolved conflict made it in.

Reply:
My last commit fix this.